pkg/diagnostics/v2/client.go

```diff
@@ -0,0 +1,43 @@
+package v2
```

Comment:
Why is this package called `v2`? The API thats called still seems to be `v1`. Is there a chance of merging this file with the already existing client.go and therefore also merge the two `List()` methods? Or are they dealing with different types of Bundles?

Reply:
It's using a new API at a slightly different endpoint which returns different types. It's pretty much entirely unrelated to the old one aside from them both talking to the same service to accomplish similar things.